bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -201,9 +207,11 @@ public static Coder getCoder(CoderType type) {
     private static final long serialVersionUID = 1L;
 
     @Override
-    public Result next(ResultScanner<Row> resultScanner) throws IOException {
+    public Result next(ResultScanner<Row> resultScanner, ByteKeyRangeTracker rangeTracker)
+        throws IOException {
       Row row = resultScanner.next();
-      if (row == null) {
+      if (row == null
```

Comment:
The [analogous code for BigtableIO](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/sdk/src/main/java/com/google/cloud/dataflow/sdk/io/bigtable/BigtableIO.java#L812).